src/book/01-team/05-robocup/03-debriefs.mdx

```diff
@@ -98,7 +102,7 @@ RoboCup Bangkok was the first in-person competition in three years due to COVID-
 #### Hardware
 
 - Find a replacement for CM740’s quick so we can buy or make them before the end of the year.
-  - Redesign NUsense to only contain the core functionality – then additional functionality can be added with shields – modular system – modules that you can plug in
+  - Redesign NUsense to only contain the core functionality. Additional functionality can be added with daughterboards/shields, creating a more modular system.
```

Comment:
Im not entirely on-board with this idea. I think the current design will do fine (apart from the hardware fixes that we know about). You can limit the code implementation to just the core functionality and add in more things later as you need/want them.

Having said that, if you do decide to do this arduino-like system you need to put in more forethought to consider space requirements of extra daughterboards/shields. How will it fit inside the torso?